### PR TITLE
Describe installation with version agnostic installer

### LINKED ISSUE #33: Update documentation for version agnostic installer
Current documentation is version specified.  As part of https://github.com/quipucords/quipucords/issues/1898 the installer now will install any version.  The documentation should be updated to:

1. Remove references to a specific version
2. Describe how to install the latest version
3. Give an example of how to install a previous version (example only)

### PR DESCRIPTION
**Pull request by kholdaway**
Closes #33

### CODE REVIEW

File: source/configure.rst [hunk 1]
```diff
@@ -60,11 +60,11 @@ Starting the Quipucords Server
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 After you make the decisions on the configuration options for the server, you can start the Quipucords server. The following commands assume that you used the default port and the recommended steps to create a home directory and subdirectories for the SSH keys, the default options for the PostgreSQL database, and the default log output during the Quipucords server configuration.
 
-If your system does not have SELinux enabled, you can start the Quipucords server with the following Docker command::
+If your system does not have SELinux enabled and you are installing release 1.0.0, you can start the Quipucords server with the following Docker command::
```
Comment (jlprevatt) [inline source/configure.rst]: If your system does not have SELinux enabled, you can start the Quipucords server with the following Docker command, where 1.0.0 is the installed version of Quipucords:

File: source/configure.rst [hunk 2]
```diff
@@ -60,11 +60,11 @@ Starting the Quipucords Server
...
 
   # sudo docker run --name quipucords --link qpc-db:qpc-link -d -e "USE_SUPERVISORD=true" -e "QPC_DBMS_USER=postgres" -e "QPC_DBMS_PASSWORD=password" -e "QPC_DBMS_HOST=qpc-db" -p 9443:443 -v ~/quipucords/sshkeys:/sshkeys -v ~/quipucords/data:/var/data -v ~/quipucords/log:/var/log -i quipucords:1.0.0
 
-If your system does have SELinux enabled, you must append ``:z`` to each volume as follows::
+If your system does have SELinux enabled and you are installing release 1.0.0, you must append ``:z`` to each volume as follows::
```
Comment (jlprevatt) [inline source/configure.rst]: If your system does have SELinux enabled, you must append ``:z`` to each volume as follows, where 1.0.0 is the installed version of Quipucords:

File: source/install_cli.rst [hunk 3]
```diff
@@ -2,39 +2,15 @@
...
-
-  - For Red Hat Enterprise Linux 7 or CentOS 7, enter the following command::
-
-      # wget -O /etc/yum.repos.d/group_quipucords-qpc-epel-7.repo https://copr.fedorainfracloud.org/coprs/g/quipucords/qpc/repo/epel-7/group_quipucords-qpc-epel-7.repo
-
-  - For Red Hat Enterprise Linux 6 or CentOS 6, enter the following command::
-
-      # wget -O /etc/yum.repos.d/group_quipucords-qpc-epel-6.repo https://copr.fedorainfracloud.org/coprs/g/quipucords/qpc/repo/epel-6/group_quipucords-qpc-epel-6.repo
-
+qpc, the command line tool that is installed by RPM, is available for `download <https://github.com/quipucords/qpc/releases/latest/>`_ from the GitHub. Use the following steps to install the command line tool.
```
Comment (myersCody) [inline source/install_cli.rst]: I don't think that the `the` is needed here. `from the GitHub` => `from GitHub`

**Review by Dostonbek1 — APPROVED** [review]
All looks good to me.

File: source/configure.rst [hunk 4]
```diff
@@ -85,7 +85,7 @@ You may also configure the quipucords server by setting the following environmen
   - DJANGO_LOGGING_LEVEL
     - Infrastructure logging level
 
-To set these values, pass the ``docker run`` command the ``-e "NAME=VALUE`` argument.  For example to set ``QPC_SERVER_TIMEOUT`` do::
+To set these values for release 1.0.0, pass the ``docker run`` command the ``-e "NAME=VALUE`` argument.  For example to set ``QPC_SERVER_TIMEOUT`` do::
```
Comment (jlprevatt) [inline source/configure.rst]: To set these values, pass the  the ``-e "NAME=VALUE`` argument for the ``docker run`` command. For example, the following command shows how to set ``QPC_SERVER_TIMEOUT`` value, where 1.0.0 is the installed version of Quipucords::

NOTE the comma after "For example" phrase
"do?" no

File: source/install_cli.rst [hunk 5]
```diff
@@ -2,39 +2,15 @@
...
-
-  - For Red Hat Enterprise Linux 7 or CentOS 7, enter the following command::
-
-      # wget -O /etc/yum.repos.d/group_quipucords-qpc-epel-7.repo https://copr.fedorainfracloud.org/coprs/g/quipucords/qpc/repo/epel-7/group_quipucords-qpc-epel-7.repo
-
-  - For Red Hat Enterprise Linux 6 or CentOS 6, enter the following command::
-
-      # wget -O /etc/yum.repos.d/group_quipucords-qpc-epel-6.repo https://copr.fedorainfracloud.org/coprs/g/quipucords/qpc/repo/epel-6/group_quipucords-qpc-epel-6.repo
-
+qpc, the command line tool that is installed by RPM, is available for `download <yum -y install https://github.com/quipucords/qpc/releases/latest/>`_ from the Fedora COPR build and repository system. Use the following steps to install the command line tool.
```
Comment (jlprevatt) [inline source/install_cli.rst]: Link says github and text still says Fedora COPR.
Also, I think Chris originally did it that way, but it is a better practice for information retrievability to *not* just highlight a word like "download" and have the link behind it. So I would prefer a structure like:
....is available for download from <full URL as link text> and <full URL as the link itself>. Use the following...

**Review by myersCody — APPROVED** [review]
I went through and tested the curls and links and it looks good to me.

File: source/troubleshooting.rst [hunk 6]
```diff
@@ -24,11 +24,11 @@ Default logging levels for the server can be altered by setting logging environm
 - DJANGO_LOGGING_LEVEL
   - Infrastructure logging
 
-If your system does not have SELinux enabled, you can start the Quipucords server with the following Docker command with increased logging::
+If your system does not have SELinux enabled and you are installing release 1.0.0, you can start the Quipucords server with the following Docker command with increased logging::
```
Comment (jlprevatt) [inline source/troubleshooting.rst]: If your system does not have SELinux enabled, you can start the Quipucords server with the following Docker command with increased logging. In the following example, the installed version of Quipucords is 1.0.0::

File: source/troubleshooting.rst [hunk 7]
```diff
@@ -24,11 +24,11 @@ Default logging levels for the server can be altered by setting logging environm
...
 
   # sudo docker run --name quipucords -d -p 9443:443 -e QUIPUCORDS_LOGGING_LEVEL=DEBUG -e DJANGO_LOGGING_LEVEL=DEBUG -v ~/quipucords/sshkeys:/sshkeys -v ~/quipucords/data:/var/data -v ~/quipucords/log:/var/log -i quipucords:1.0.0
 
-If your system does have SELinux enabled, you must append ``:z`` to each volume as follows::
+If your system does have SELinux enabled and you are installing release 1.0.0, you must append ``:z`` to each volume as follows::
```
Comment (jlprevatt) [inline source/troubleshooting.rst]: If your system does have SELinux enabled, you must append ``:z`` to each volume as follows. In the following example, the installed version of Quipucords is 1.0.0::

File: source/configure.rst (hunk 1 above)
Comment (kholdaway) [reply source/configure.rst]: 👍

File: source/configure.rst (hunk 2 above)
Comment (kholdaway) [reply source/configure.rst]: 👍

File: source/configure.rst (hunk 4 above)
Comment (kholdaway) [reply source/configure.rst]: 👍

File: source/install_cli.rst (hunk 3 above)
Comment (kholdaway) [reply source/install_cli.rst]: Now you are going to tell me it's not the Facebook

File: source/troubleshooting.rst (hunk 6 above)
Comment (kholdaway) [reply source/troubleshooting.rst]: 👍

File: source/troubleshooting.rst (hunk 7 above)
Comment (kholdaway) [reply source/troubleshooting.rst]: 👍

File: source/install_cli.rst (hunk 5 above)
Comment (kholdaway) [reply source/install_cli.rst]: Gotcha.

**Pull request merged**